apps/server/src/shared/domain/entity/school.entity.ts

```diff
@@ -129,11 +130,15 @@ export class SchoolEntity extends BaseEntity {
 			this.features = props.features;
 		}
 		if (props.schoolYear) {
-			this.schoolYear = props.schoolYear;
+			this.currentYear = props.schoolYear;
 		}
 		if (props.userLoginMigration) {
 			this.userLoginMigration = props.userLoginMigration;
 		}
 		this.federalState = props.federalState;
+		// TODO: General question: Why the if-checks around the optional props?
```

Comment:
@CeEv noch mal bei task / user schauen. Irgendwas mit default Werte / undefined explizit setzten oder so, in bestimmten Situationen.